instructions/docs/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+## Background
+
+In this section, you will prepare your AWS Environment to work with the Busy Engineer's Document Bucket. You will:
+
+1. Deploy a CloudFormation stack to launch a Cloud9 IDE
+1. Launch the Cloud9 IDE that you will use to work with the rest of the workshop
+1. Launch resource stacks using the AWS Cloud Development Kit (CDK), including
+    * The `BusyEngineersDocumentBucket` stack, with your DynamoDB table and S3 bucket
+    * One CMK in one region, called `Faythe`, that you will use in encryption and decryption operations
+    * One CMK in another region, called `Walter`, that you will use in encryption and decryption operations
+1. Bootstrap the development environment in Cloud9
+1. Start the workshop!
+
+## Make the Change
+
+1. Sign in to your AWS Account for the workshop
+    * Make sure this is **not** a production account!
```

Comment:
why not?

Reply:
Hm, let me try to strike a better balance of not-alarmism and attempts at good advice.